src/NServiceBus.NHibernate.PersistenceTests/NServiceBus.NHibernate.PersistenceTests.csproj

```diff
@@ -14,7 +14,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
     <PackageReference Include="NHibernate" Version="5.2.7" />
-    <PackageReference Include="NServiceBus.PersistenceTests.Sources" Version="7.4.2" />
+    <PackageReference Include="NServiceBus.PersistenceTests.Sources" Version="8.0.0-alpha.541" />
```

Comment:
This should probably by .554?

Reply:
Yes, you're right. This was changed before we released the latest alpha. I'll fix it.